apps/web/next.config.mjs

```diff
@@ -1,7 +1,7 @@
 /* eslint-disable @typescript-eslint/no-var-requires */
 import BundleAnalyzer from '@next/bundle-analyzer'
 import { withWebSecurityHeaders } from '@pancakeswap/next-config/withWebSecurityHeaders'
-import smartRouterPkgs from '@pancakeswap/smart-router/package.json' with { type: 'json' }
+import smartRouterPkgs from '@pancakeswap/smart-router/package.json' assert { type: 'json' }
```

Comment:
any reason for the update here?

Reply:
@thechefpenguin, could you take a look at this?

Reply:
I guess this is about the node version

Reply:
@chefjackson I get an eslint error when using `with`. 
> Parsing error: ';' expected.  

That perhaps for static json imports we should use `assert`.  Not a major issue so I can revert back to `with` if we want

Reply:
let's revert back. I think we changed that due to higher nodejs version updated this convention